js/src/offcanvas.js

```diff
@@ -102,7 +102,11 @@ class Offcanvas extends BaseComponent {
     }
 
     this._isShown = true
-    this._backdrop.show()
+    this._element.classList.add("offcanvas-end") // Adding the class 'offcanvas-end' here
```

Comment:
This feels wrong, every `offcanvas` isn't an `offcanvas-end`.

Reply:
My pardon.

show(relatedTarget) {
    if (this._isShown) {
        return;
    }

    const showEvent = EventHandler.trigger(this._element, EVENT_SHOW, { relatedTarget });

    if (showEvent.defaultPrevented) {
        return;
    }

    this._isShown = true;

    // Adding the class 'offcanvas-end' here
    this._backdrop.show();

    if (!this._config.scroll) {
        new ScrollBarHelper().hide();
    }

    this._element.setAttribute('aria-modal', true);
    this._element.setAttribute('role', 'dialog');
    this._element.classList.add(CLASS_NAME_SHOWING);

    const completeCallBack = () => {
        if (!this._config.scroll || this._config.backdrop) {
            this._focustrap.activate();
        }

        this._element.classList.add(CLASS_NAME_SHOW);
        this._element.classList.remove(CLASS_NAME_SHOWING);
        EventHandler.trigger(this._element, EVENT_SHOWN, { relatedTarget });
    };

    // Introduce a small delay before triggering the Offcanvas
    setTimeout(completeCallBack, 50); // Adjust the delay as needed
}    

Please suggest what should be done!